contracts/JBETHPaymentTerminalStore.sol

```diff
@@ -80,6 +101,14 @@ contract JBETHPaymentTerminalStore {
   // --------------------- public stored properties -------------------- //
   //*********************************************************************//
 
+  /** 
+    @notice
+    Maximum redemption rate of the funding cycle.
+  */
+  uint256 constant MAX_REDEMPTION_RATE = 10000;
```

Comment:
should be using the new JBConstants instead now?

Reply:
I only created that for common constants, so the localized constants are in the individual files